pandas/core/arrays/timedeltas.py

```diff
@@ -288,6 +314,21 @@ def _evaluate_with_timedelta_like(self, other, op):
 
         return NotImplemented
 
+    __mul__ = _wrap_tdi_op(operator.mul)
+    __rmul__ = __mul__
```

Comment:
Why's `rmul` have a different pattern?

Reply:
Because multiplication is commutative.  Or am I misunderstanding the question?